libs/service-portal/vehicles/src/screens/VehicleDetail/VehicleDetail.tsx

```diff
@@ -0,0 +1,271 @@
+import React from 'react'
+
+import {
+  Box,
+  Divider,
+  GridColumn,
+  GridRow,
+  Table as T,
+  Stack,
+  Text,
+  LoadingDots,
+} from '@island.is/island-ui/core'
+import {
+  ServicePortalModuleComponent,
+  UserInfoLine,
+} from '@island.is/service-portal/core'
+import { useLocale, useNamespaces } from '@island.is/localization'
+
+import BaseInfoItem from '../../components/DetailTable/BaseInfoItem'
+import RegistrationInfoItem from '../../components/DetailTable/RegistrationInfoItem'
+import OwnerInfoItem from '../../components/DetailTable/OwnerInfoItem'
+import InspectionInfoItem from '../../components/DetailTable/InspectionInfoItem'
+import TechnicalInfoItem from '../../components/DetailTable/TechnicalInfoItem'
+import OwnersTable from '../../components/DetailTable/OwnersTable'
+import { gql, useQuery } from '@apollo/client'
+import { CurrentOwnerInfo, Query } from '@island.is/api/schema'
+import OperatorInfoItem from '../../components/DetailTable/OperatorInfoItem'
+import CoOwnerInfoItem from '../../components/DetailTable/CoOwnerInfoItem'
+
+const GET_USERS_VEHICLE_DETAIL = gql`
```

Comment:
Would be nice to clear up the components and make a specific gql query folder for these.

Reply:
✅